src/Angor/Client/Shared/NavMenu.razor

```diff
@@ -174,16 +206,6 @@
 
     }
 
-    /// <inheritdoc />
-    protected override async Task OnInitializedAsync()
-    {
-        _networkService.AddSettingsIfNotExist();
```

Comment:
where did this code go?

Reply:
![image](https://github.com/block-core/angor/assets/6504337/dabfb9df-9d0e-4838-9fb8-64907c8a060d)